packages/replay/README.md

```diff
@@ -87,6 +89,46 @@ replay.start(); // Start recording
 replay.stop(); // Stop recording
 ```
 
+## Loading Replay as a CDN bundle
+
+As an alternative to the NPM package, you can load the Replay Integration bundle from our CDN.
+Note that the Replay bundle **only contains the Replay integration** and not the entire Sentry SDK.
+You have to add it in addition to the Sentry Browser SDK bundle:
+
+```js
+// Browser SDK bundle
+<script
+  src="https://browser.sentry-cdn.com/7.24.0/bundle.min.js"
```

Comment:
Is this the bundle including tracing, or without? IMHO we should specify the one with tracing here, as I guess that is the "default"/"recommended" option?

Reply:
Yup, good point. Right now it's just the browser bundle but I'll change it

Reply:
[2ab6381](https://github.com/getsentry/sentry-javascript/pull/6431/commits/2ab6381c0ee2cd929d8482b64f12bc7dcda442fe)